chapter09/mdw/Ex05.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+$^I= ".out";
+
+my $line = 1;
+
+while (<>) {
+	if ( $line == 2 ) {
+		if ( !(/\#\# Copyright \(C\) 2015 by mdw/) ){
+			s/(.*)/## Copyright (C) 2015 by mdw\n$1/;
+		};
```

Comment:
So I'm assuming the extra <code>;</code> after <code>if</code> block is to say there is no <code>else</code> block?